examples/animation/animated_fox.rs

```diff
@@ -0,0 +1,127 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .insert_resource(AmbientLight {
+            color: Color::WHITE,
+            brightness: 1.0,
+        })
+        .add_startup_system(setup)
+        .add_system(setup_scene_once_loaded)
+        .add_system(keyboard_animation_control)
+        .run();
+}
+
+struct Animations(Vec<Handle<AnimationClip>>);
+
+fn setup(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    mut scene_spawner: ResMut<SceneSpawner>,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    // Insert a resource with the current scene information
+    commands.insert_resource(Animations(vec![
+        asset_server.load("models/animated/Fox.glb#Animation2"),
+        asset_server.load("models/animated/Fox.glb#Animation1"),
+        asset_server.load("models/animated/Fox.glb#Animation0"),
+    ]));
+
+    // Camera
+    commands.spawn_bundle(PerspectiveCameraBundle {
+        transform: Transform::from_xyz(100.0, 100.0, 150.0)
+            .looking_at(Vec3::new(0.0, 20.0, 0.0), Vec3::Y),
+        ..Default::default()
+    });
+
+    // Plane
+    commands.spawn_bundle(PbrBundle {
+        mesh: meshes.add(Mesh::from(shape::Plane { size: 500000.0 })),
+        material: materials.add(Color::rgb(0.3, 0.5, 0.3).into()),
+        ..default()
+    });
+
+    // Light
+    commands.spawn_bundle(DirectionalLightBundle {
+        transform: Transform::from_rotation(Quat::from_euler(
+            EulerRot::ZYX,
+            0.0,
+            1.0,
+            -std::f32::consts::FRAC_PI_4,
+        )),
+        directional_light: DirectionalLight {
+            shadows_enabled: true,
+            ..default()
+        },
+        ..default()
+    });
+
+    // Fox
+    scene_spawner.spawn(asset_server.load("models/animated/Fox.glb#Scene0"));
+
+    println!("Animation controls:");
+    println!("  - spacebar: play / pause");
+    println!("  - arrow up / down: speed up / slow down animation playback");
+    println!("  - arrow left / right: seek backward / forward");
+    println!("  - return: change animation");
+}
+
+// Once the scene is loaded, start the animation
+fn setup_scene_once_loaded(
+    animations: Res<Animations>,
+    mut player: Query<&mut AnimationPlayer>,
+    mut done: Local<bool>,
+) {
+    if !*done {
+        if let Ok(mut player) = player.get_single_mut() {
+            player.play_animation(animations.0[0].clone_weak()).repeat();
```

Comment:
Hm... Looking from a use perspective AnimationPlayer would look prettier if it was just Animation.
system param
animations: Query<&mut Animation>,
then later
if let Ok(mut animation) = animations.get_single_mut() {
animation.play(fox_animations.0[0].clone_weak()).repeat();

Reply:
I do like the bias toward "nicer names", but in this particular case, I think AnimationPlayer is the better name. It represents a thing that plays animation, not a thing that _is_ Animation. I'm more open to renaming AnimationClip to Animation, but I'm also happy with the current name, for now.

Additionally, we have plans for a more general "bevy animation system". It probably makes sense to reserve the Animation name until we sort out how that fits in to this.